### PR TITLE
Codeberg & Mastodon

### PR DESCRIPTION
**Pull request by mefuller**
Added Codeberg (https://codeberg.org/) icon to socials (one more major git server for project hosting).
Added some minor tidying on the template

EDIT: also added Mastodon since it's also in FontAwesome and trivial to implement

N.B. the footer on the template is overflowing now, but no one would use _all_ the icons at once (I hope)

### CODE REVIEW

File: template.tex [hunk 1]
```diff
@@ -54,20 +54,27 @@
...
+\social[stackoverflow]{0000000/johndoe}            % optional, remove / comment the line if not wanted
+
 \social[skype]{jdoe}                               % optional, remove / comment the line if not wanted
 \social[orcid]{0000-0000-000-000}                  % optional, remove / comment the line if not wanted
 \social[researchgate]{jdoe}                        % optional, remove / comment the line if not wanted
 \social[researcherid]{jdoe}                        % optional, remove / comment the line if not wanted
+\social[googlescholar]{googlescholarid}            % optional, remove / comment the line if not wanted
+
+\social[twitter]{ji\_doe}                          % optional, remove / comment the line if not wanted
+\social[mastodon]{mastodon.social/web/@user}       % optional, remove / comment the line if not wanted
```
Comment (stephanlachnit) [inline template.tex:71]: I guess the reason why we don't add the full link here is because that mastodon is federated?

**Comment by stephanlachnit** [comment]
Thanks, I like it!

File: template.tex (hunk 1 above)
Comment (mefuller) [reply template.tex:71]: Yes - mastodon.social is just one of many possibilities (e.g. I am at fosstodon.org)

**Comment by stephanlachnit** [comment]
Merged now for `2.4.0` 👍🏼

**Pull request merged**